src/Ng/Jobs/Catalog2LuceneJob.cs

```diff
@@ -107,6 +107,13 @@ protected override void Init(IDictionary<string, string> arguments, Cancellation
             }
         }
 
+        ReadCursor GetTheLeastAdvancedRegistrationCursor(string registrationArg, CancellationToken cancellationToken)
+        {
+            string[] registrations = registrationArg.Split(';');
```

Comment:
After offline discussion, I'll file an issue regarding how we can harden our services to change with index.json changes.